core/services/ocr2/plugins/ocr2keeper/evm21/transmit/event_provider.go

```diff
@@ -208,18 +202,12 @@ func (c *TransmitEventProvider) processLogs(latestBlock int64, logs ...logpoller
 				UpkeepID:        *upkeepId,
 				CheckBlock:      trigger.BlockNumber,
 			}
+			c.cache.add(k, transmitEvent)
```

Comment:
Just checking - adding to cache here means the confirmations won't be stored on the event in the cache, is that okay?